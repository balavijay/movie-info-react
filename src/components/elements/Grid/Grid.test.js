import React from 'react';
import { shallow } from 'enzyme';


import Grid from './Grid';

describe ("Grid Component " , () => {
   const  app = shallow(<Grid />);
    test("Component render" , () => {
        expect(app).toMatchSnapshot();
    })

    

});