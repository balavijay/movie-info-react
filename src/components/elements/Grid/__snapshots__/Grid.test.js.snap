// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid Component  Component render 1`] = `ShallowWrapper {}`;
